src/modules/sdl2/consumer_sdl2_audio.c

```diff
@@ -593,8 +597,10 @@ static void *consumer_thread(void *arg)
             // Play audio
             init_audio = consumer_play_audio(self, frame, init_audio, &duration);
 
-            // Determine the start time now
-            if (self->playing && init_video) {
+            // Start the video thread unconditionally. We cannot check for self->playing,
+            // because the variable may be initialized with a delay by a backdround SDL
+            // thread and cause race condition
+            if (init_video) {
```

Comment:
Since `playing` is now only set and never read, it should be removed along with the comment here that no longer makes sense.